lib/tasks/import/service_interactions.rake

```diff
@@ -32,5 +34,10 @@ namespace :import do
     task add_service_tiers: :environment do
       LocalLinksManager::Import::ServicesTierImporter.import
     end
+
+    desc "Enable services used on Gov.uk"
+    task enable_all: :environment do
```

Comment:
Could we rename the task to something like `enable_services`? Right now because of the namespacing it sounds like we're enabling all ServiceInteractions instead of Services.